regression/goto-analyzer/ternary-operator-value-sets/test.desc

```diff
@@ -3,10 +3,10 @@ main.c
 --variable-sensitivity --vsd-values set-of-constants --show
 ^EXIT=0$
 ^SIGNAL=0$
-main::1::b1 \(\) -> value-set-begin: TRUE, :value-set-end
```

Comment:
:+1: Good to see the formatting being improved and the tests being made more stable.